www/base/src/styles/styles.less

```diff
@@ -121,19 +121,12 @@ Firefox will insert newlines if the selection go over a no-select span, so we ne
   padding-right: 10px;
 }
 
-.avatar {
-  background-size: 32px 32px;
+img.avatar {
```

Comment:
Is `div > img` for IE? I'm not sure why the border radius wasn't placed directly on the image and I'm assuming this is just to support older IE, but I'm assuming I didn't notice any prefixes or they aren't being generated anymore.

Reply:
We no longer support IE at all. Edge 13 is the minimum (according to https://github.com/buildbot/buildbot/blob/master/www/base/src/app/app.browserwarning.notranspile.js)